platform_channels/lib/src/method_channel_demo.dart

```diff
@@ -0,0 +1,88 @@
+// Copyright 2020 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:platform_channels/src/counter_method_channel.dart';
+
+/// The widget demonstrates how to use [MethodChannel] to invoke platform methods.
+/// It has two [FloatingActionButton]s and a [Text] widget to display the value of
+/// [count].
+class MethodChannelDemo extends StatefulWidget {
+  @override
+  _MethodChannelDemoState createState() => _MethodChannelDemoState();
+}
+
+class _MethodChannelDemoState extends State<MethodChannelDemo> {
+  int count;
+  final GlobalKey<ScaffoldState> _scaffoldKey = GlobalKey<ScaffoldState>();
+
+  @override
+  void initState() {
+    super.initState();
+    count = 0;
```

Comment:
You can also just initialize count to 0 right in its declaration and get ride of this override.